Configuration/PyReleaseValidation/python/relval_nano.py

```diff
@@ -244,6 +250,9 @@ def subnext(self):
 steps['scoutingNANO_data14.0'] = merge([{'-s': 'NANO:@Scout'},
                                         steps['NANO_data14.0']])
 
+steps['scoutingNANO_withPrompt_data14.0'] = merge([{'-s': 'NANO:@Prompt+@Scout', '--customise_commands': '"process.options.TryToContinue = cms.untracked.vstring(\'ProductNotFound\')"'},
```

Comment:
Why does it need the `process.options.TryToContinue = cms.untracked.vstring(\'ProductNotFound\')` here?

Reply:
Thank you for checking!

Some events in ScoutingPFMonitor do not have scouting objects and it will crash because the products cannot be found.

To solve this, we can put default value instead. For SimpleFlatTableProducer, we can set skipNonExistingSrc=True. However, there is no similar option in GlobalVariablesTableProducer which is used for ScoutingMET and ScoutingRho. Also, while it is reasonable to put empty array in SimpleFlatTableProducer when product (e.g. jets, muons) is not found, it is not clear to me what value to use when MET or Rho is not found. We can put -1, but option to put default value when product is not found will need to implement for GlobalVariablesTableProducer.

Then, skipping the events without scouting objects seems better than keeping them and use default values. It should be possible with EDFilter. I couldn't find an EDFilter that selects event based on whether product(s) are valid, so this needs to be implemented. This issue (some events do not have scouting objects) should be resolved for 2025 data taking in [CMSHLT-3331](https://its.cern.ch/jira/browse/CMSHLT-3331). So we may want to use this EDFilter only for 2024 (and maybe in 2022-23), but not in 2025.

So, here, I just use the easiest option: ```process.options.TryToContinue = cms.untracked.vstring(\'ProductNotFound\')```.

Reply:
Thanks a lot for the explanation! Since the problem is already solved for 2025, I think this is OK then. I would only suggest adding a comment about this in the code, then we can sign off the PR.

Reply:
Thanks! I just added the comment.

When I was writing the explanation, we can also try EDFilter approach later. For now, I think it is ok since the problem should be solved for 2025.